index.html

```diff
@@ -99,7 +99,7 @@ <h4>Workshop.</h4>
         <div class="program">
           <i class="far fa-handshake fa-2x"></i>
           <h4>E Ignite!</h4>
-          <p>Get opportunities to network with E-learning affliate around the world, also 
+          <p>Get opportunity to network with E-learning affliate around the world, also 
             after the summit. Great Networking Event</p>
         </div>
       </div>
```

Comment:
* Kindly fix the menu bar as the menu bar on mobile doesn't appear 
* Also please fix the errors in the console i get this error when i scroll down the page 
![image](https://user-images.githubusercontent.com/52497246/133490771-7002b5d6-8278-4a35-9bb6-24bb6d3d7237.png)